package.json

```diff
@@ -12,8 +12,9 @@
   ],
   "scripts": {
     "build": "webpack -p",
-    "build:dev": "webpack --progress --colors --mode=development",
     "build:demo": "webpack --config ./demo/webpack.config.js",
+    "build:dev": "webpack --progress --colors --mode=development",
+    "dev": "npm run build:dev -- --watch",
```

Comment:
Intentionally going in with this change?

Reply:
Oh, not 100% intentional. I do want the change to go in, but it's not tightly connected to this feature. I'll make a separate PR and cherry pick it.